pom.xml

```diff
@@ -4,29 +4,212 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>demo.mockito</groupId>
-    <artifactId>mockito-demo</artifactId>
+    <groupId>mockito-demo-public</groupId>
+    <artifactId>mockito-demo-public</artifactId>
     <version>1.0-SNAPSHOT</version>
 
+    <properties>
+        <maven.compiler.target>17</maven.compiler.target>
+        <maven.compiler.source>17</maven.compiler.source>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <file.path>/Users/sachinkumar/Desktop/cli_demo/mockito-demo-public/.sapient/cli_config.txt</file.path>
+        <key1>projectRoot</key1>
+        <new.value1>${project.basedir}</new.value1>
+        <key2>moduleName</key2>
+        <new.value2>${project.name}</new.value2>
+        <key3>testingFramework</key3>
+        <new.value3>JUnitPlatformTestFramework</new.value3>
+        <key4>testLibraryVersion</key4>
+        <new.value4>5.9</new.value4>
+        <key5>testJavaSrcDir</key5>
```

Comment:
can we do it a way we did for gradle, ex: move the these to an external maven file?